app/assets/stylesheets/application.css

```diff
@@ -12,4 +12,6 @@
  *
```

Comment:
- [ ] It would be nice if you make tests for the controllers and you know that controllers and views are one of the most crucial components of your application if they don't behave as expected means your application is not working as expected so it would be better for your application if you test your controllers and views :+1: